cico_functions.sh

```diff
@@ -33,6 +33,24 @@ function load_jenkins_vars() {
   fi
 }
 
+function check_buildx_support() {
+  docker_version="$(docker --version | cut -d' ' -f3 | tr -cd '0-9.')"
+  if [[ "$(version "$docker_version")" < "$(version '19.03')" ]]; then
```

Comment:
In my experience, CI systems will rarely have the latest version of docker. Do we know if these changes work on our CI as it stands? Should there be a fallback in case `check_buildx_support()` fails, to proceed with the build as normal?

Reply:
> In my experience, CI systems will rarely have the latest version of docker.

Our changes are based on the assumption that CI system will use latest docker and support buildx.

> Do we know if these changes work on our CI as it stands?

We started with docker buildx support activity as suggested by @benoitf in this [comment](https://github.com/eclipse/che/issues/16287#issuecomment-633560366)

> Should there be a fallback in case check_buildx_support() fails, to proceed with the build as normal?

If we have to fallback on normal build, that will not require any need of updating docker version in CI and this activity will not give us expected results. Hence, multiarch images won't be produced.

Reply:
Looking at recent CI runs, it looks like we're now updating Docker using the docker-ce repo before running the build, so my comment appears to be irrelevant :)